crates/fuel-core/src/executor.rs

```diff
@@ -1053,7 +1064,10 @@ where
                             )?;
                             if tx_pointer != &coin.tx_pointer {
                                 return Err(ExecutorError::InvalidTransactionOutcome {
-                                    transaction_id: tx.id(),
+                                    transaction_id: tx.id(&self
```

Comment:
Maybe we can calculate it one time at the begin of the `execute_create_or_script` method and pass it everywhere?

Reply:
Can do as a follow up